tictactoe.py

```diff
@@ -7,6 +7,37 @@ def game_instructions():
         print(instructions)
 
 
+class Board:
+    def __init__(self):
+        self.board = [["1", "2", "3"], ["4", "5", "6"], ["7", "8", "9"]]
+        self.markers = {"X": None, "O": None}
+        self.players = []
+
+    def create_player(self, name):
+        self.players.append(Player(name))
+
+    def assign_players(self):
+        # assigns players to X and O
+        self.markers["X"] = self.players[0]
+        self.markers["O"] = self.players[1]
+
+    def draw_board(self):
```

Comment:
When we think about the Single Responsibility Principle here, is this a sign that our `Board` class is now doing more than one thing? If we switched out our eventual output (in our case, `stdout`) for a GUI and wanted to retain our current Board class (starting to consider the Open/Closed principle) - this `draw_board` function would potentially be misleading (it would not be able to render the GUI) or unused.